Library/CharacterSet.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+public extension CharacterSet {
```

Comment:
I've considered using `private extension` but in order for us to test this I think this is a better approach..